src/main/java/com/google/sps/servlets/MealServlet.java

```diff
@@ -187,24 +204,23 @@ private Meal pickRandomMeal(List<Meal> mealList) {
         return (Meal)mealList.get(index);
     }
 
-    private Boolean isResultOfSearch(Meal meal, List<String> params) {
+    private int isResultOfSearch(Meal meal, List<String> params) {
```

Comment:
It's kind of strange that `isResultOfSearch` returns `int` value. Probably we need to rename it.